cmd/gpu_plugin/gpu_plugin_test.go

```diff
@@ -314,3 +317,110 @@ func TestScan(t *testing.T) {
 		})
 	}
 }
+
+// Would be nice to combine these with the overall Scan unit tests.
```

Comment:
But it's not done because?

To find this comment, it would be good to mark it as TODO or FIXME (or do the work, or drop the comment).